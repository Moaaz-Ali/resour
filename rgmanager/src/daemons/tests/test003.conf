<?xml version="1.0"?>
<!-- Add an initscript to our regular resource group.
     The initscript should get a 'NEEDSTART' flag. -->
<cluster>
<rm>
	<resources>
    		<resourcegroup name="test1"/>
		<script name="initscript" file="/etc/init.d/httpd"/>
	</resources>
	<resourcegroup ref="test1">
		<script ref="initscript"/>
	</resourcegroup>
</rm>
</cluster>
