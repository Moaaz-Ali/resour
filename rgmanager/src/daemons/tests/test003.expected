=== Resources List ===
Resource type: script
Agent: script.sh
Attributes:
  name = initscript [ primary unique ]
  file = /etc/init.d/httpd [ unique required ]

Resource type: resourcegroup [ROOT]
Instances: 1/1
Agent: resourcegroup.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
resourcegroup {
  name = "test1";
  script {
    name = "initscript";
    file = "/etc/init.d/httpd";
  }
}
